### PR TITLE
Countries Widget

### PR DESCRIPTION
**Pull request by mihaelabuda**

### CODE REVIEW

File: js/homepage.js [hunk 1]
```diff
@@ -4,13 +4,20 @@ var towns = ["Amsterdam", "London", "Paris", "Copenhagen", "Madrid", "Rome", "St
 var randomTown = Math.floor(Math.random() * 10);
 var randomInfo = towns[randomTown];
 
-$(document).ready(function widgetWeather() {
+$(document).ready(function widget() {
```
Comment (scozma) [inline js/homepage.js]: @mihaelabuda - Please move the two 'ajax' blocks into their own special functions and call them inside the 'widget' function (maybe rename this to 'populateWidgets' or 'generateWidgets', or something similar)

File: js/homepage.js [hunk 2]
```diff
@@ -4,13 +4,20 @@ var towns = ["Amsterdam", "London", "Paris", "Copenhagen", "Madrid", "Rome", "St
...
     $.ajax({
```
Comment (scozma) [inline js/homepage.js]: @mihaelabuda - Please add 'error' properties to the ajax response object. When you get an error, you should display a message to the bottom right corner about this, using the existing notification/error component.

File: js/homepage.js (hunk 1 above)
Comment (mihaelabuda) [reply js/homepage.js]: Hi @scozma - Changes applied.

File: js/homepage.js (hunk 2 above)
Comment (mihaelabuda) [reply js/homepage.js]: Hi @scozma - Changes applied.

**Pull request merged**